### PR TITLE
Added welcome page for meetup pre-talk

### PR DESCRIPTION
**Pull request by AverageMarcus**
An editable page to show on the projector before talks. Can show the speaker lineup and wifi details.

More logos can be added to the JS Oxford logo so if anyone has any suggestions let me know.

### CODE REVIEW

**Comment by benfoxall** [comment]
super ace.  One thing that might make it easier to style is to use `contenteditable` rather than `textarea` which is nice because the size can be a bit more flexble and you can make things bold/italic.

```html
<span contenteditable></span>
```

…and this css will allow you to see/click empty spans I think

```css
[contenteditable] {min-width:1em;display:inline-block}
[contenteditable]:hover {background:rgba(255,255,255,0.3)}
```

File: welcome.html [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+
+  </ul>
+
+  <div class="jo-wifi">
+    <strong>WiFi</strong> <span contenteditable></span> / <span contenteditable></span>
+  </div>
+</div>
+
+  <script>
+    var numberOfSpeakers = 2; //prompt("How many speakers do we have today?", 2);
```
Comment (benfoxall) [inline welcome.html]: oh, I think you left in this debugging code - was it supposed to prompt?

Comment (AverageMarcus) [inline welcome.html]: dammit!

Comment (AverageMarcus) [inline welcome.html]: Gone now.

**Pull request merged**